promscale/manage-data/downsampling.md

```diff
@@ -0,0 +1,259 @@
+```
+
+Now you can query the average without the `__column__` label, like this:
+```promql
+node_memfree_1hour
+```
+
+Promscale adds a `__schema__` tag, which helps you identify in which schema the metric view is registered. This is useful if you have two metrics with the same name in different schemas, which is strongly discouraged but could happen by mistake. In this case, you can use the `__schema__` tag to select which schema to use. If you don't specify a schema, Promscale queries the metric in the `prom_data` schema, which is the same schema used for all ingested metrics. To avoid those issues, make sure you name your continuous aggregate views in a way that is different to your raw ingested metrics, for example, `node_memfree_1hour`.
+
+<highlight type="note">
+Both the `__schema__` and `__column__` label matchers support exact matching. You cannot use regular expressions or other multi-value matchers. Additionally, metric views are excluded from queries that match multiple metrics, such as matching on metric names with a regular expression, like this:
+
+```sql
+{__name__=~"node_mem*"} // this valid PromQL query will not match our previously created metric view
+```
+
+</highlight>
+
+### Deleting continuous aggregates
+To delete a Promscale continuous aggregate, delete the metric view first, and
+then remove the continuous aggregate.
+
+Remove the metric view:
+```sql
+unregister_metric_view('public', 'node_memfree_1hour');
+```
+
+Delete the continuous aggregate:
+```sql
+DROP MATERIALIZED VIEW node_memfree_1hour;
+```
+
+## Prometheus downsampling with recording rules
+Prometheus provides a downsampling mechanism called recording rules. These rules
+are used to calculate frequently used or computationally expensive queries
+before they are required. A recording rule is a PromQL expression that
+Prometheus evaluates at a predefined frequency to generate a new metric series.
+The new metric series is stored in Promscale, and you can query it in the same
+way as any other Prometheus metric. An example recording rule:
+```sql
+groups:
+  - name: daily_stats
+    interval: 1h
+    rules:
+    - record: customer:api_requests:rate1day
+      expr: sum by (customer) (rate(api_requests_total[1d]))
+
+```
+
+For the recording rules to be applied, add the location of the recording rules
+file to the Prometheus configuration file:
+```yaml
+rule_files:
+  - "<recording-rules-file>"
+```
+
+<highlight type="note">
+When you set up the Prometheus configuration file to use recording rules, we
+recommend that you also set `read_recent` to `true` in the
+[Prometheus remote_read configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read).
+This tells Prometheus to fetch data from Promscale when evaluating PromQL
+queries, including recording rules. If `read_recent` is disabled, only the data
+stored in the local TimescaleDB instance is used when evaluating alerting and
```

Comment:
This originally said "data stored in Prometheus's local tsdb". Tsdb in this context is not TimescaleDB, it's how the local timeseries database is named in Prometheus: https://prometheus.io/docs/prometheus/latest/storage/ (search for tsdb).

To disambiguate, I suggest we say something like "stored in Prometheus' local on-disk time series database"

Reply:
```suggestion
stored in the Prometheus local on-disk database is used when evaluating alerting and
```